modules/ROOT/pages/depl-examples/bare-metal.adoc

```diff
@@ -73,26 +73,38 @@ Note that this URL must resolve to the server running this installation.
 
 === Install and configure the Infinite Scale Binary
 
-* Download and install the Infinite Scale binary. To get the stable binary from ownCloud, visit {ocis-downloadpage-url}/{download-type}/?sort=time&order=desc[download.owncloud.com,window=_blank], select the version and the platform of choice and copy the link of the file.
+* Download and install the Infinite Scale binary. To get the stable binary from ownCloud, visit {ocis-downloadpage-url}/{download-type}/?sort=time&order=desc[download.owncloud.com,window=_blank], select the version and the platform of choice and copy the link of the file. Replace `<file_url>` and `<version>` in the commands accordingly.
 +
+--
+Download the source:
+
 [source,bash,subs="attributes+"]
 ----
-sudo wget -O {ocis_bin}/ocis <file_url>
+sudo wget -P {ocis_bin} <file_url>
 ----
+--
 
 * Make the binary executable with:
 +
 [source,bash,subs="attributes+"]
 ----
-sudo chmod +x {ocis_bin}/ocis
+sudo chmod a+x {ocis_bin}/ocis-<version>
+----
+
+* Create a link from the source to the final executable named `ocis`: +
+Note when using a link to the source, upgrading and/or testing is much easier.
```

Comment:
I am puzzled. One of a sudden, the binary is called source. Do I need to compile something now?
```suggestion
Note when using a symbolic link like this, upgrading and/or testing is much easier.
```